.kiro/specs/technology-stack-decisions/design.md

```
@@ -0,0 +1,884 @@
+  | 'garden' | 'graveyard' | 'courtyard';
+```
+
+## Correctness Properties
+
+*A property is a characteristic or behavior that should hold true across all valid executions of a system - essentially, a formal statement about what the system should do. Properties serve as the bridge between human-readable specifications and machine-verifiable correctness guarantees.*
+
+### Property 1: Morph Target Round-Trip Consistency
+
+*For any* valid morph weight configuration applied to the DAZ Genesis model, saving the character state and loading it back SHALL produce an identical visual appearance.
+
+**Validates: Requirements 2.3, 2.5**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)